InnerEye/ML/configs/histo_configs/classification/DeepSMILECrck.py

```diff
@@ -87,6 +90,22 @@ def cache_dir(self) -> Path:
             f"/tmp/innereye_cache1/{self.__class__.__name__}-{self.encoder_type}/"
         )
 
+    def setup(self) -> None:
+        if self.encoder_type == InnerEyeSSLEncoder.__name__:
+            self.downloader = CheckpointDownloader(
+                azure_config_json_path=get_workspace(),
+                run_id="updated_transforms:updated_transforms_1636471522_5473e3ff",
+                checkpoint_filename="best_checkpoint.ckpt",
+                download_dir="outputs/",
+                remote_checkpoint_dir=Path("outputs/checkpoints")
+            )
+            os.chdir(fixed_paths.repository_root_directory().parent)
+            self.downloader.download_checkpoint_if_necessary()
```

Comment:
you have defined repository_root_parent already, re-use that?
Also, why is it necessary to have a side effect here, and change working directory?